### ISSUE TITLE
[Bug]: 502 Bad Gateway on V1 2.2.0

### ISSUE THREAD

**Issue by wellhat** [acheong08/ChatGPT#735]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits
- 
My apologies if this is not a bug in 2.2.2. I was only able to get 2.2.0 using `pip3 install revChatGPT`.

### What happened?

Since V2 is down for maintenance today I switched my implementation to V1, but when calling chatbot.ask("Hello ChatGPT"), I got a Bad Gateway response. I am using a paid account.

I created a ChatbotSingleton:

```
class ChatbotSingleton:
    _instance = None

    def __new__(cls, *args, **kwargs):
        if not cls._instance:
            print("Logging in...")
            chatbot = Chatbot(config={
            "email": "email@mail.com",
            "password": "snip"
            })
            # V2 currently not working
            # chatbot = Chatbot(email="email@mail.com", password="snip", paid=True)
            print("Logged in")
            cls._instance = chatbot
        return cls._instance

```

```
print("Client said: " + message)
for data in chatbot_instance.ask(
            message,
            conversation_id=chatbot_instance.config.get("conversation"),
            parent_id=chatbot_instance.config.get("parent_id"),
            ):
    print(data["message"], end="", flush = True)
print("ChatGPT responding: " + data["message"])
self.write_message(data["message"])
```

The output is:

```
Client said: Hello ChatGPT
<html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.18.0 (Ubuntu)</center>
</body>
</html>
```

I was able to get the app working in V2 yesterday with my credentials and similar setup.

### Steps to reproduce the problem

Run the code with an initial message "Hello ChatGPT"

### What should have happened?

ChatGPT response is received successfully

### Version where the problem happens

python3 -m pip show revChatGPT

Name: revChatGPT
Version: 2.2.0
Summary: ChatGPT is a reverse engineering of OpenAI's ChatGPT API
Home-page: https://github.com/acheong08/ChatGPT
Author: Antonio Cheong
Author-email: acheong@student.dalat.org
License: GNU General Public License v2.0
Location: /Users/jh/Library/Python/3.9/lib/python/site-packages
Requires: asyncio, httpx, OpenAIAuth, requests, tiktoken
Required-by: 

### What Python version are you running this with?

Python 3.9.6 

### What is your operating system ?

MacOS

### Command Line Arguments

```Shell
No
```

### Console logs

```Shell
w@My-MacBook-Air ChatGPTApp % python3 myapp.py
Listening for connections on port 8888
Logging in...
Logged in to email@mail.com

Client said: Hello ChatGPT
<html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
<hr><center>nginx/1.18.0 (Ubuntu)</center>
</body>
</html>
```

**Comment by acheong08** [comment]
Update to https://github.com/acheong08/ChatGPT/releases/tag/2.2.2

**Comment by acheong08** [comment]
`pip3 install --upgrade revChatGPT`

**Comment by wellhat** [comment]
Great, thanks @acheong08

**Comment by wellhat** [comment]
On 2.2.2 I get the error:

```
Field missing
{'detail': {'message': 'An error occurred. Try refreshing your browser.', 'code': 'model_not_found'}}
```

I will open a new issue if I don't find a duplicate & some debugging doesn't uncover a user error

**Comment by acheong08** [comment]
Downgrade to `2.2.1` for now then `pip3 install revChatGPT==2.2.1`

**Comment by wellhat** [comment]
That did the trick @acheong08, thanks for your help.
